plugins/org.eclipse.fordiac.ide.systemmanagement.ui/plugin.xml

```diff
@@ -588,7 +588,28 @@
             </instanceof>
          </filterExpression>
       </commonFilter>
-  </extension>  
+  </extension>
+      <extension
+            point="org.eclipse.ui.navigator.navigatorContent">
+         <navigatorContent
+               activeByDefault="true"
+               id="org.eclipse.fordiac.ide.systemmanagement.ui.systemexplorer.sortOnly"
+               name="name"
```

Comment:
This should be a sensible human readable text because it is shown in the user interface.

Reply:
change name to something reasonable